docs/src/TextArea.doc.js

```diff
@@ -44,16 +44,24 @@ card(
       },
       {
         name: 'onBlur',
-        type: '({ event: SyntheticFocusEvent<>, value: string }) => void',
+        type:
```

Comment:
These are going to cause Flow errors when bumping in pinboard - prepare yourselves, winter is coming